src/components/TodoListItem/TodoListItem.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import styles from './TodoListItem.module.css';
+
+const TodoListItem = ({ todo, onRemoveTodo }) => {
+  return (
+    <React.Fragment>
+      <div className={styles.TodoItem}>
+        <div>
+          <input id={todo.id} type="checkbox" />
+          <label htmlFor={todo.id}>{todo.title}</label>
+        </div>
+        <div>
+          <button
+            type="button"
+            className={` ${styles.remove}`}
```

Comment:
Since it's only taking one style you don't need this template literal

```suggestion
            className={styles.remove}
```

Reply:
Updated.